ir_attachment_conversion_queue_job/models/ir_attachment.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+
+class Irattachemnt(models.Model):
+    _inherit = "ir.attachment"
+
+    def _image_job_queue(self):
+        mapping = {
+            "product.template": "image",
+            "product.image": "image",
+            "product.public.category": "image",
+            "res.partner": "image",
+            "res.users": "image",
+        }
+        for model, field in mapping.items():
+            attachments = self.search(
+                [
+                    ("res_model", "=", model),
+                    ("res_field", "=", field),
+                    ("res_id", "!=", False),
+                ]
+            )
+            for attachment in attachments:
+                self.with_delay()._convert_image_attachments(model, attachment)
+
+    def _convert_image_attachments(self, model, attachment):
+        try:
+            self.env[model].browse(attachment.res_id).image_1920 = attachment.datas
+        except Exception as e:
+            _logger.error(
+                "Error while recovering for %s: %s",
+                attachment.res_id,
+                repr(e),
+            )
```

Comment:
Is there actually any point doing `try` and `except` in a job?

```suggestion
        self.env[model].browse(attachment.res_id).image_1920 = attachment.datas
```